postBuild.js

```diff
@@ -35,6 +36,10 @@ async function moveComponentsFiles() {
   });
 }
 
+async function moveFontFiles() {
+  fs.copySync(`src/fonts`, `${buildDir}/fonts`);
```

Comment:
I think would be better to put font to Theme package, it will allow to use it from the outside as well

Reply:
updated!